### PR TITLE
fix(route/ieee): fix journal.ts, author.ts and earlyaccess.ts

### PR DESCRIPTION
**Pull request by Derekmini**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/ieee/journal/8782710
/ieee/journal/4427201/earlyaccess
/ieee/author/37281702200
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [ ] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
1. Update the method from `got` to `ofetch` in `/ieee/journal.ts` and `/ieee/earlyaccess.ts`;
2. The route `routes/ieee/recent.ts` has been deprecated.

### CODE REVIEW

**Comment by Derekmini** [comment]
May I ask how to solve the following problem? Is it currently a network-related issue?

> Test / Vitest puppeteer on Node 20 with Chrome from Google (pull_request) 
> ❯ lib/utils/puppeteer-utils.test.ts  (5 tests | 1 failed) 23821ms
>    ❯ lib/utils/puppeteer-utils.test.ts > puppeteer-utils > getCookies httpbingo
>      → Test timed out in 20000ms.

**Comment by Derekmini** [comment]
> http://localhost:1200/ieee/author/37281702200 - Failed ❌
> ```
> HTTPError: Response code 404 (Not Found)
> 
> Error Message:<br/><code class="mt-2 block max-h-28 overflow-auto bg-zinc-100 align-bottom w-fit details">NotFoundError: The route does not exist or has been deleted.</code>
> Route: <code class="ml-2 bg-zinc-100">/ieee/author/37281702200</code>
> Full Route: <code class="ml-2 bg-zinc-100">/ieee/author/37281702200</code>
> Node Version: <code class="ml-2 bg-zinc-100">v21.7.2</code>
> Git Hash: <code class="ml-2 bg-zinc-100">40fe6e7e</code>
> ```

This problem is related to assets/build/router.json, I don't know how to update this file.

**Comment by Derekmini** [comment]
timer

File: lib/routes/ieee/author.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+const __dirname = getCurrentPath(import.meta.url);
+
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import path from 'node:path';
+import { art } from '@/utils/render';
+
+export const route: Route = {
+    path: '/author/:aid/:sortType/:count?',
```
Comment (lewisjiang) [inline lib/routes/ieee/author.ts:12]: This should be `path: '/author/:aid/:sortType?/:count?',` since sortType is not mandantory.

Comment (Derekmini) [reply lib/routes/ieee/author.ts]: thanks for your reviewed.

**Comment by Derekmini** [comment]
\

**Pull request closed without merging**